decidim-assemblies/spec/system/admin/admin_manages_assembly_soft_delete_spec.rb

```diff
@@ -12,4 +12,26 @@
 
   it_behaves_like "manage soft deletable component or space", "assembly"
   it_behaves_like "manage trashed resource", "assembly"
+
+  context "when a user is collaborator" do
```

Comment:
I guess this happens for all the user types ( moderators, evaluators, Space admins) . Maybe you could add a more specs for each one of the roles?